Makefile.OCaml: fswatch.cmi depends on ubase/prefs.cmi.

  The dependency is needed to compile unison
  in parallel.

Reference: https://github.com/bcpierce00/unison/pull/100

diff --git a/Makefile.OCaml b/Makefile.OCaml
index 21610ce..84fe7ad 100644
--- a/Makefile.OCaml
+++ b/Makefile.OCaml
@@ -426,6 +426,8 @@ win32rc/unison.res.lib: win32rc/unison.res
 	@echo "$(CAMLC): $< ---> $@"
 	$(CAMLC) $(CAMLFLAGS) -c $(CWD)/$<
 
+fswatch.cmi : ubase/prefs.cmi
+
 %.cmo: %.ml
 	@echo "$(OCAMLC): $< ---> $@"
 	$(OCAMLC) $(CAMLFLAGS) -c $(CWD)/$<
